### ISSUE TITLE
input the end of codeblock 

### ISSUE THREAD

**Issue by jakekwak** [Vanessa219/vditor#705]
Can not see the next line when enter the end of input area. (typewriterMode=false)

![vditor23](https://user-images.githubusercontent.com/15811943/89790070-05555380-dae7-11ea-9847-15adef6fa80b.gif)

**Comment by Vanessa219** [comment]
Yes, this mode only show the cursor. If the next line is displayed, you can set `typewriterMode = true`.

**Comment by jakekwak** [comment]
Is it possible?
![vditor25](https://user-images.githubusercontent.com/15811943/89858162-8c481180-db63-11ea-8887-7183c4aae5b3.gif)

**Comment by Vanessa219** [comment]
The typewriterMode event must be triggered after the press Enter, maybe we also need to trigger it after pressing arrowXXX?

**Comment by jakekwak** [comment]
Thanks, 
src/ts/ir/index.ts line:170
```
            if (event.key === "Enter" || event.key === "ArrowDown") {
                scrollCenter(vditor);
            }
```
